<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>capy::ui::ColorArea</class>
    <widget class="QWidget" name="capy::ui::ColorArea">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>643</width>
                <height>657</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>Selector:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QComboBox" name="selectorComboBox"/>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="Line" name="line_2">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="addColorToPalette">
                    <property name="text">
                        <string>Add color to current palette</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="DefaultColorPicker" name="colorPickerWidget" native="true"/>
            </item>
            <item>
                <widget class="Line" name="line_3">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>Palette:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="paletteComboBox"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <item>
                                <widget class="QPushButton" name="createPaletteButton">
                                    <property name="text">
                                        <string>Create</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="removePaletteButton">
                                    <property name="text">
                                        <string>Delete</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="Line" name="line">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QTableView" name="colorTableView"/>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                        <widget class="QPushButton" name="createColorButton">
                            <property name="text">
                                <string>Add color</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="removeColorButton">
                            <property name="text">
                                <string>Delete color</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>DefaultColorPicker</class>
            <extends>QWidget</extends>
            <header>ui/widgets/color-pickers/DefaultColorPicker.hpp</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
